src/status_im/utils/ethereum/stateofus.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.utils.ethereum.stateofus
+  (:require [clojure.string :as string]))
+
+(def domain "stateofus.eth")
+
+(defn- subdomain [username]
+  (str username "." domain))
+
+(def registrars
```

Comment:
do we actually need this?

`(ens/owner "0x112234455c3a32fd11230c42e7bccd4a84e02010" "stateofus.eth" println)`
0x11d9f481effd20d76cee832559bd9aca25405841

Reply:
Actually it's something we need to check statically and pretty often so doesn't appear to be worth the hassle and perf cost.

Reply:
what do you mean we need to check it statically? what if we change the contract address for stateofus.eth?

Reply:
Well same as if we change any contract address?

Reply:
@decanus do you have any recommendation here? Should we rather rely on hardcoded contract addresses for the registrar or solve it with the ens resolver contract?